test/support/test_config.h

```diff
@@ -23,6 +23,26 @@
 #   include <ciso646>
 #endif
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// This section contains macros representing the "Latest" version of compilers, STL implementations, etc. for use in
+// broken macros to represent the latest version of something which still has an ongoing issue. The intention is to
+// update this section regularly to reflect the latest version. 
+//
+// *** When updating we must audit each usage to ensure that the issue is still exists in the latest version ***

```

Comment:
I think this line should be moved either to the start of the comment block or the end of the comment block. As it is currently, it might be easy to miss.

Reply:
Moved it to the top.  Thanks.